modules/ui/OptimizerParamsWindow.py

```diff
@@ -142,7 +142,7 @@ def create_dynamic_ui(
             'r': {'title': 'R', 'tooltip': 'EMA factor.', 'type': 'float'},
             'adanorm': {'title': 'AdaNorm', 'tooltip': 'Whether to use the AdaNorm variant', 'type': 'bool'},
             'adam_debias': {'title': 'Adam Debias', 'tooltip': 'Only correct the denominator to avoid inflating step sizes early in training.', 'type': 'bool'},
-
+            'slice_p': {'title': 'Slice parameters', 'tooltip': 'Reduce memory usage by calculating LR adaptation statistics on only every pth entry of each tensor. For values greater than 1 this an an approximation to standard Prodigy. Values ~11 are reasonable.', 'type': 'int'},
```

Comment:
I recommend that we change our tooltip to mention this, and also bring in their improved reasoning [from their readme](https://github.com/konstmish/prodigy?tab=readme-ov-file#how-to-use) about why 11 is good. Also improved the grammar a bit:

- `'slice_p': {'title': 'Slice parameters', 'tooltip': 'Reduce memory usage by calculating LR adaptation statistics only for the p-th values of each tensor. For values greater than 1, this is an approximation of standard Prodigy. Value should always be uneven to avoid skipping the same channels every time; 11 is a good trade-off between learning rate estimation and memory efficiency.', 'type': 'int'},`

Reply:
imho it is an expert _opinion_ that uneven numbers *might* work better. We've never shown that it actually is better. All my tests showing that it works were with slice_p == 10.
Could be something for expert users in the Wiki